docs/en/revocation_endpoint.rst

```diff
@@ -0,0 +1,72 @@
+.. include:: ./common_definitions.rst
+
+Revocation Endpoint (logout)
+----------------------------
+
+An RP CAN request the revocation of an Access Token or a Refresh Token issued by an OP.
+
+When the user logs out, the RP MUST revoke the Access Token in its possession.
+
+.. note::
+  The Access Token revocation implies revoking all the Refresh Tokens linked to it.
+
+The OP WILL HAVE TO revoke the token specified in the request and WILL HAVE TO end the Single Sign-On session, if it is still active.
```

Comment:
Do we have WILL HAVE TO in the normative lagnuage section? If not please change with a defined normative language